provider/fs/src/export/aliasing.rs

```diff
@@ -36,8 +37,8 @@ pub(crate) struct AliasCollection<T> {
     symlink_file_extension: String,
     data_file_prefix: String,
     data_file_extension: String,
-    map: HashMap<T, Vec<PathBuf>>,
-    flushed: bool,
+    // An empty option means the collection has been flushed to disk.
+    map: Mutex<Option<HashMap<T, Vec<PathBuf>>>>,
 }
 
```

Comment:
Optional: If it makes things easier, feel free to delete this file and the symlink aliasing features.  This was always fraught.